magda-web-client/src/AppContainer.js

```diff
@@ -34,6 +34,7 @@ import mobileMenu from "./assets/mobile-menu.svg";
 import { Route, Link, Switch } from "react-router-dom";
 
 import "./AppContainer.css";
+import DatasetDetails from "./Components/Dataset/DatasetDetails";
```

Comment:
Is this needed? "  37:8  warning  'DatasetDetails' is defined but never used  no-unused-vars"

BTW if you run a tool called eslint or associate it with your editor it will warn you about some problems https://circleci.com/gh/TerriaJS/magda/73